pipeline.py

```diff
@@ -72,31 +89,79 @@ class TextProcessingPipeline:
     """
 
     def __init__(self, corpus_manager: CorpusManager):
+        self.corpus_manager = corpus_manager
         pass
 
     def run(self):
         """
         Runs pipeline process scenario
         """
-        pass
+        for article_item in self.corpus_manager.get_articles().values():
+            article_text = article_item.get_raw_text()
+            article_text = article_text.replace('-\n', '')
+            for symbol in ['\n', '\r']:
+                article_text = article_text.replace(symbol, ' ')
+            re_tokens = re.findall(r'[а-яё_]+-?[а-яё_]*', article_text, flags=re.IGNORECASE)
+            cleaned_text = ' '.join(re_tokens)
+            article_item.save_as(cleaned_text.lower(), 'cleaned')
```

Comment:
save of article should happen after you run processing. Otherwise it breaks the overall logic